CHANGELOG.md

```diff
@@ -1,12 +1,20 @@
 Changes from v1.5 to v2.0
 =========================
 
+Dependency Changes
+------------------
+
+- Added LSSTDESC.Coord. (#809b)
+
+
 API Changes
 -----------
 
-
-Dependency Changes
-------------------
+- There were some minor API changes to the Angle and CelestialCoord classes we made when we
+  moved it over into LSSTDESC.Coord.  Some were to sever (weak) ties to other GalSim classes and
+  some were just deemed API improvements.  Probably the most relevant of these for most
+  users will be that Angle.rad is now a property rather than a function.  So the value of an
+  angle in radians is just `theta.rad` not `theta.rad()`. (#809b)
```

Comment:
Huh, I could see that being a pain.  No way to be backwards compatible, or is the philosophy here that going to v2 is a big enough change that people should expect instantaneous API changes anyway?